Formula/jetty.rb

```diff
@@ -4,6 +4,7 @@ class Jetty < Formula
   url "https://search.maven.org/remotecontent?filepath=org/eclipse/jetty/jetty-distribution/9.4.32.v20200930/jetty-distribution-9.4.32.v20200930.tar.gz"
   version "9.4.32.v20200930"
   sha256 "64e5fe203347a4b4f5b5a828f9c241b2f13f154b56babccd533513991fd156c2"
+  license "Apache-2.0"
```

Comment:
```suggestion
  license any_of: ["Apache-2.0", "EPL-1.0"]
```

From the readme:

> Jetty is open source and is dual licensed using the Apache 2.0 and
> Eclipse Public License 1.0.   You may choose either license when
> distributing Jetty.

Reply:
ty, just figured out the dual license myself as well